ProjectSettings/TagManager.asset

```diff
@@ -59,5 +59,5 @@ TagManager:
     uniqueID: 1439066835
     locked: 0
   - name: Layer 5
-    uniqueID: 387849965
+    uniqueID: 214891783
```

Comment:
Can you revert this diff to the TagManager.asset file? You'll want to make sure that your minigame doesn't break without this.

Reply:
I have reverted back the file. Please review this PR again.